.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,61 @@
+name: Bug report
+description: Create a report to help us improve
+title: "[BUG] <title>"
+labels: ["bug"]
+projects: ["livechat/7"]
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+        - label: I have searched the existing issues
+          required: true
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps To Reproduce
+      description: Steps to reproduce the behavior.
+      placeholder: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See an error
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: A clear and concise description of what you expected to happen.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Screenshots
+      description: If applicable, add screenshots to help explain your problem.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Environment
+      description: |
+        examples:
+          - **Design system package(s) version**: [e.g. @livechat/design-system-react-components@1.0.0-beta.11]
+          - **Device**: [e.g. iPhone6]
+          - **OS**: [e.g. macOS]
+          - **Browser**: [e.g. chrome, safari]
+          - **Version**: [e.g. 22]
+      render: markdown
+    validations:
+      required: false
```

Comment:
IMO the package version is also important and should be required. Maybe it should be a separate field?

Reply:
Good point. I'll add it separately.